exonum-java-binding/core/rust/integration_tests/tests/service_runtime.rs

```diff
@@ -17,14 +17,100 @@
 extern crate exonum_testkit;
 extern crate integration_tests;
 extern crate java_bindings;
+#[macro_use]
+extern crate lazy_static;
+extern crate parking_lot;
 
 use exonum_testkit::TestKitBuilder;
-use integration_tests::vm::{get_fake_service_artifact_path, get_fakes_classpath, get_libpath};
+use integration_tests::{
+    fake_service::*,
+    vm::{get_fake_service_artifact_path, get_fakes_classpath, get_libpath},
+};
 use java_bindings::{Config, JavaServiceRuntime, JvmConfig, RuntimeConfig};
+use parking_lot::{Mutex, MutexGuard};
+
+lazy_static! {
+    // Reference to the JavaServiceRuntime is wrapped with Mutex in order to execute tests (more
+    // precise - Java code for artifacts loading and services creation) sequentially.
+    // We use Mutex from the `parking_lot` crate because it doesn't get poisoned in case of panic.
+    static ref SERVICE_RUNTIME: Mutex<JavaServiceRuntime> = Mutex::new(create_runtime());
+}
+
+#[test]
+fn load_one_service() {
+    let runtime = get_runtime();
+    let artifact_path = create_service_artifact_valid(runtime.get_executor());
+    let artifact_id = runtime.load_artifact(&artifact_path);
+    let service = runtime.create_service(&artifact_id);
+
+    let mut testkit = TestKitBuilder::validator().with_service(service).create();
+
+    testkit.create_block();
+}
 
 #[test]
-fn bootstrap() {
-    let artifact_path = get_fake_service_artifact_path();
+fn load_two_services() {
+    let runtime = get_runtime();
+
+    let fake_artifact_path = get_fake_service_artifact_path();
+    let fake_artifact_id = runtime.load_artifact(&fake_artifact_path);
+    let fake_service = runtime.create_service(&fake_artifact_id);
+
+    let valid_artifact_path = create_service_artifact_valid(runtime.get_executor());
+    let valid_artifact_id = runtime.load_artifact(&valid_artifact_path);
+    let valid_service = runtime.create_service(&valid_artifact_id);
+
+    let mut testkit = TestKitBuilder::validator()
+        .with_service(fake_service)
+        .with_service(valid_service)
+        .create();
+
+    testkit.create_block();
+}
+
+#[test]
+#[should_panic(expected = "Unable to load artifact")]
+fn load_nonexistent_artifact() {
+    let runtime = get_runtime();
+    let artifact_path = "nonexistent_artifact.jar";
+    runtime.load_artifact(&artifact_path);
+}
+
+#[test]
+#[should_panic(expected = "Unable to load artifact")]
+fn load_artifact_twice() {
+    let runtime = get_runtime();
+    let artifact_path = create_service_artifact_valid(runtime.get_executor());
+
+    runtime.load_artifact(&artifact_path);
```

Comment:
This test is especially susceptible to this issue: https://errorprone.info/bugpattern/TestExceptionChecker